rocAL/rocAL/source/meta_data/bounding_box_graph.cpp

```diff
@@ -29,6 +30,46 @@ void BoundingBoxGraph::process(MetaDataBatch *meta_data)
     }
 }
 
+//update_meta_data is not required since the bbox are normalized in the very beggining -> removed the call in master graph also except for MaskRCNN
+void BoundingBoxGraph::update_meta_data(MetaDataBatch *input_meta_data, decoded_image_info decode_image_info)
```

Comment:
Refer https://github.com/SundarRajan28/MIVisionX/commit/a43f7ac11796870e9b4e1f01bd1a94404193ef5d and https://github.com/SundarRajan28/MIVisionX/commit/11126d6bbf649bfbc69425f2888527f1195891a8